src/humanPose/HumanPoseAnalyzerSettingsUi.js

```diff
@@ -28,6 +28,10 @@ export class HumanPoseAnalyzerSettingsUi {
                             <div class="hpa-settings-section-row-label">View auxiliary poses</div>
                             <input type="checkbox" class="hpa-settings-section-row-checkbox" id="hpa-settings-toggle-child-human-poses">
                         </div>
+                        <div class="hpa-settings-section-row">
+                            <div class="hpa-settings-section-row-label">Set joint confidence</div>
+                            <input type="text" id="hpa-settings-set-joint-confidence">
```

Comment:
input type="number" could be a better choice here (might need to create some css styles to make it fit in)